modules/install-ibm-cloud-configuring-the-public-subnet.adoc

```diff
@@ -0,0 +1,138 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-public-subnet_{context}"]
+= Configuring the public subnet
+
+All of the {product-title} cluster nodes must be on the public subnet. IBM Cloud&#174; does not provide a DHCP server on the subnet. Set it up separately on the provisioner node.
+
+.Procedure
+
+. Install `dnsmasq`:
++
+[source,terminal]
+----
+$ sudo dnf install dnsmasq
+----
+
+. Set an unused IP address from the public IP range:
++
+[source,terminal]
+----
+$ UNUSEDIP=<IP-address>
+----
+
+. Open the `dnsmasq` configuration file:
++
+[source,terminal]
+----
+$ sudo vi /etc/dnsmasq.conf
+----
+
+. Add the following configuration to the `dnsmasq` configuration file:
++
+[source,configuration files]
+----
+interface=baremetal
+except-interface=lo
+bind-dynamic
+log-dhcp
+
+dhcp-range=$UNUSEDIP,$UNUSEDIP,$PUBLICCIDR
+dhcp-option=baremetal,121,0.0.0.0/0,$BAREMETAL_GATEWAY,$BOOTSTRAP_PRIV_IP,$BSHOSTIP
+
+dhcp-hostsfile=/var/lib/dnsmasq/dnsmasq.hostsfile
+----
++
+[NOTE]
+====
+The Dnsmasq configuration uses the BASH variables created when preparing the provisioner node. If you install the DHCP server on a node other than the provisioner node, you will need to re-create these BASH variables.
+====
```

Comment:
After 'init 6' in previous instructions "Set up the bridges for the `baremetal` and `provisioning` networks" all BASH variables should be re-created